app/params/proto.go

```diff
@@ -0,0 +1,22 @@
+package params
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+)
+
+// MakeEncodingConfig creates an EncodingConfig for an amino based test configuration.
+func MakeEncodingConfig() EncodingConfig {
```

Comment:
please use the existing `MakeEncodingConfig()` in `app/encoding.go` so we can avoid code duplication